src/arb.h

```diff
@@ -638,7 +638,7 @@ arb_sqr(arb_t res, const arb_t val, slong prec)
 
 /* vector functions */
 
-ARB_INLINE arb_ptr
+FLINT_FORCE_INLINE arb_ptr
 _arb_vec_entry_ptr(arb_ptr vec, slong i)
```

Comment:
This is an example of a function that people call externally and which should thus remain in the library.